packages/tracing-internal/src/browser/request.ts

```diff
@@ -275,22 +275,19 @@ export function xhrCallback(
   }
 
   const scope = getCurrentScope();
-  const parentSpan = getActiveSpan();
-
-  const span =
-    shouldCreateSpanResult && parentSpan
-      ? // eslint-disable-next-line deprecation/deprecation
-        parentSpan.startChild({
-          data: {
-            type: 'xhr',
-            'http.method': sentryXhrData.method,
-            url: sentryXhrData.url,
-          },
-          description: `${sentryXhrData.method} ${sentryXhrData.url}`,
-          op: 'http.client',
-          origin: 'auto.http.browser',
-        })
-      : undefined;
+
+  const span = shouldCreateSpanResult
+    ? startInactiveSpan({
```

Comment:
I guess there is no easy way to make these active spans 🤔 They aren't now either, so I guess it's OK, but just thinking, for the future!

Reply:
I think we don't want to make them active spans because they are async, it's not ideal, but something we can come back to later on.